### ISSUE TITLE
5318008 Error code

### ISSUE THREAD

**Issue by Ronizlo** [MyShiLingStar/ACNHPokerCore#76]
I can not seem to download Sys-bot correctly. My home button does not glow when powered on, and 5318008 error code keeps popping up when I press connect.

Please help me install Sys-bot correctly.

**Comment by MyShiLingStar** [comment]
Please refer to https://github.com/MyShiLingStar/ACNHPokerCore/issues/73 and see if that helps.
If not, please post the screenshots of your SD card.
Especially at `SD: \ atmosphere \ contents`

**Comment by Ronizlo** [comment]
![Screenshot 2024-02-25 012517](https://github.com/MyShiLingStar/ACNHPokerCore/assets/161200968/58ebf81f-15dd-4c39-b092-67ee387e9a53)
This is what is in my Folder

**Comment by Ronizlo** [comment]
![Screenshot 2024-02-25 012517](https://github.com/MyShiLingStar/ACNHPokerCore/assets/161200968/998d66ab-1634-4c08-a478-4e9dd2bc4302)

**Comment by Ronizlo** [comment]
I know it can't be my internet, I have done everything it says when the error screen pops up, and my switch has "AMS" on it.

**Comment by MyShiLingStar** [comment]
![image](https://github.com/MyShiLingStar/ACNHPokerCore/assets/816502/e376bd5e-66b1-4084-8f39-7c1d62e39498)
No `sys-botbase-2.31` folder at all.

Please fix your folder structure so that it goes like this.
`SD: \ atmosphere \ contents \ 430000000000000B \`

**Comment by Ronizlo** [comment]
I can't have the atmosphere folder there though because I have another atmosphere which is what my CFW is running off of.
![image](https://github.com/MyShiLingStar/ACNHPokerCore/assets/161200968/ba27cb04-9407-4e92-b609-e4c6b02a14f1)

**Comment by MyShiLingStar** [comment]
You can. They are the same entities. The one and only.
Just merge them so that they could become whole once again.
They were not meant to be separated.

**Comment by Ronizlo** [comment]
So you're saying just move the atmosphere folder to the old one?

**Comment by Ronizlo** [comment]
Please help.

**Comment by MyShiLingStar** [comment]
Yes, please move and merge them so that they can stay together forever.
Only one `atmosphere` folder should exist on the SD card. Only at the root `SD: \ atmosphere`.
You have installed something wrong if you find another `atmosphere` folder elsewhere.

**Comment by Ronizlo** [comment]
The other atmosphere folder was from downloading the actual atmosphere application for my CFW.

**Comment by MyShiLingStar** [comment]
![image](https://github.com/MyShiLingStar/ACNHPokerCore/assets/816502/39a9eb6c-5485-4a0a-9dca-663cc8461f5c)
Please do not create another `atmosphere` folder inside the `atmosphere` folder.
`SD: \ atmosphere  \ atmosphere` is incorrect.

**Comment by Ronizlo** [comment]
Sorry I'm confused, what do you mean by merging them? I thought you just meant add the Sys-Botbase atmosphere into the other atmosphere folder. Sorry, please explain.

**Comment by Ronizlo** [comment]
But if I did this, Atmosphere would be completely removed, wouldn't it?

**Comment by MyShiLingStar** [comment]
Nope, a single `atmosphere` folder would still exist inside the SD card.
Since you merge two `atmosphere` folders into one single `atmosphere` folder.

**Comment by Ronizlo** [comment]
Ohh, I see. Let me try this and let you know if it works.

**Comment by Ronizlo** [comment]
So when I merge the two, do I remove the atmosphere folder that was already in my SD card?
